tests/functional/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -327,6 +327,7 @@ class TestNodeReplacementTwice(ManageTest):
       2. ceph side host still on the old rack
     """
 
+    @skipif_ibm_cloud_managed
```

Comment:
Instead of skipping, why not fix it in ocs-ci?

Reply:
My guess is that in IBM ROKS managed platform the node operations are very limited and this might be the case here that this is not possible to implement on IBM CLOUD Managed AKA ROKS platform. But @vasudha786 know better.